### DISCUSSION TITLE
Importing a skin

### DISCUSSION THREAD

**Discussion by Mochified** [ppy/osu#23395] [Q&A]
I've been trying to import a skin into osu!lazer on android and everything says about pressing the import button but I just don't know what to do after that, so many folders show up and I don't even see the osk (not sure if thats what I should even be looking for). So my question is, is there anyone who can give somewhat detailed steps on how to import a skin from start to finish if possible?

**Comment by Zero001235** [comment]
It doesn't working on latest updates

**Reply by Mochified** [reply]
Not working? Dang it. Thank you for your reply!

**Comment by sineplusx** [comment]
If you cant use the "Import files" menu  in game, you can still import skins by using your file manager. File managers can vary greatly in layouts and functionality, so I cant really give you a detailed description.

For me, a click on the skin file is enough to trigger the import when the game is already running in the background.

**Comment by Zero001235** [comment]
On android and ios it doesn't work, I don't know why, but after fix 2023.326 it doesn't work

**Comment by peppy** [comment]
Hard to confirm 100%, but probably this issue which is being tracked as a regression https://github.com/ppy/osu/issues/23106.

**Comment by MineTurtlee** [comment]
For me, you can get the skin file, then drag it to lazer window.
